docs/getting-started/detections-req.asciidoc

```diff
@@ -74,24 +73,32 @@ image::images/sec-admin-user.png[]
 [[access-detections-ui]]
 == Access and use Detections
 
-After enabling Detections, only users with these permission can view and use the
-*Detections* page:
+After enabling Detections, only users with these permissions can view and use rules and alerts on the *Detections* page:
 
-* {kib} space `All` privileges for the `Security`  and `Saved Objects
-Management` features
+**All**
+
+These permissions are required for both rule and alert management:
+
+* {kib} space with `All` privileges enabled for `Security`.
```

Comment:
In `7.12` we can probably remove this in favor of calling it out below in the `**Rule**` and `**Alert**` sections. Rule will stay as `All`, and `Alerts` will be `Read`.

Reply:
I'll make a separate PR for this comment, once I've backported this one.